charts/values.yaml

```diff
@@ -117,3 +117,16 @@ startupProbe:
   failureThreshold: 30
   successThreshold: 1
 
+logging:
+  enabled: false
+  trackFiles:
```

Comment:
Another naming suggestion: I think you can drop "track" from this and just call it "files". So this setting would be "logCollection.files".